7c7f7b2a86fe845ce0385757c1d07a7ea787b8b3
apollo-portal/src/test/java/com/ctrip/apollo/portal/repository/AppRepositoryTest.java
--- 
+++ 
@@ -1,28 +1,28 @@
 package com.ctrip.apollo.portal.repository;
+
+import com.ctrip.apollo.portal.AbstractPortalTest;
+import com.ctrip.apollo.portal.entity.App;
 
 import org.junit.Assert;
 import org.junit.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 
-import com.ctrip.apollo.portal.AbstractPortalTest;
-import com.ctrip.apollo.portal.entity.App;
 
-
-public class AppRepositoryTest extends AbstractPortalTest{
+public class AppRepositoryTest extends AbstractPortalTest {
 
   @Autowired
   AppRepository repository;
 
   @Test
   public void testCreate() {
     Assert.assertEquals(0, repository.count());
 
     App ramdomApp = new App();
     ramdomApp.setAppId(System.currentTimeMillis());
     ramdomApp.setName("new app " + System.currentTimeMillis());
     ramdomApp.setOwner("owner " + System.currentTimeMillis());
     repository.save(ramdomApp);
 
     Assert.assertEquals(1, repository.count());
   }
 }